CommunityProject/Assets/_Game/Scripts/Runtime/Enemies/ScriptableObjects/EnemySO.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+using UnityEngine.Localization;
+
+namespace BoundfoxStudios.CommunityProject.Enemies.ScriptableObjects
+{
+	/// <summary>
+	/// Base Information about an enemy
+	/// </summary>
+	[CreateAssetMenu(menuName = Constants.MenuNames.Enemies + "/Enemy")]
+	public class EnemySO : ScriptableObject
+	{
+		[field: SerializeField]
+		public LocalizedString Name { get; private set; }
+
+		[field: SerializeField, Min(1)]
```

Comment:
Ich würd gerne, auch wenn's sehr verbose ist in der Schreibeweise, pro Attribut eine eigene Zeile machen. So ein kleines wie `Min(0)` geht schnell mal runter.

```suggestion
		[field: SerializeField]
		[field: Min(1)]
```

Entsprechend dann bei den anderen bitte auch :)